site-src/guides/index.md

```diff
@@ -255,33 +255,17 @@ This quickstart guide is intended for engineers familiar with k8s and model serv
 
    Wait until the gateway is ready.
 
-=== "GPU-Based Model Server"
-
-      ```bash
-      IP=$(kubectl get gateway/inference-gateway -o jsonpath='{.status.addresses[0].value}')
-      PORT=80
-
-      curl -i ${IP}:${PORT}/v1/completions -H 'Content-Type: application/json' -d '{
-      "model": "food-review",
-      "prompt": "Write as if you were a critic: San Francisco",
-      "max_tokens": 100,
-      "temperature": 0
-      }'
-      ```
-
-=== "CPU-Based Model Server"
-
-      ```bash
-      IP=$(kubectl get gateway/inference-gateway -o jsonpath='{.status.addresses[0].value}')
-      PORT=80
-
-      curl -i ${IP}:${PORT}/v1/completions -H 'Content-Type: application/json' -d '{
-      "model": "Qwen/Qwen2.5-1.5B-Instruct",
-      "prompt": "Write as if you were a critic: San Francisco",
-      "max_tokens": 100,
-      "temperature": 0
-      }'
-      ```
+   ```bash
+   IP=$(kubectl get gateway/inference-gateway -o jsonpath='{.status.addresses[0].value}')
+   PORT=80
+
+   curl -i ${IP}:${PORT}/v1/completions -H 'Content-Type: application/json' -d '{
+   "model": "food-review-1",
```

Comment:
PTAL at the [InferenceModel manifest](https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/config/manifests/inferencemodel.yaml) used by the quickstart guide. The curl request should target the `food-review` model. The EPP will change the model name based on the `food-review` InferenceModel configuration.

Reply:
@danehans I wasn’t looking in the InferenceModel resources, so I suggested food-review-1. you’re right 👍🏻